File: PRAK405-2510817320004-Rahma-Syarita-Jaya.c
#include <stdio.h>

int main(){
    int maks, angka, hasil_akhir = 0;
    scanf("%d %d", &maks, &angka);
    
    for(int i = 1; i <= maks; i++){
        int hasil = 0;
        for (int j = i; j >= 1; j--){
            hasil += j * angka;
            printf("(%d * %d)", j, angka);
                if(j - 1 != 0){
                    printf(" + ");
                } else{
                    printf(" = %d\n", hasil); 
                }
        } 
        hasil_akhir += hasil;
    }
    printf("%d", hasil_akhir);  
    return 0;
}

File: PRAK405-2510817320004-Rahma-Syarita-Jaya.py
maks, angka = map(int, input().split())
hasil_akhir = 0

for i in range(maks + 1):
    hasil = 0
    for j in range(i, 0, -1):
        hasil += j * angka
        print(f"({j} * {angka})", end=" ")
        if j != 1:
            print("+", end=" ")
        else:
            print(f"= {hasil}")
    hasil_akhir += hasil
print(f"{hasil_akhir}")